electronicparsers/wien2k/parser.py

```diff
@@ -1053,7 +1096,44 @@ def parse_method(self):
             sec_method.k_mesh = sec_k_mesh
             sec_k_mesh.points = kpoints[0]
             sec_k_mesh.weights = kpoints[1]
-
+        if self.inc_parser.mainfile:
+            self.inc_parser.parse()
+            atoms = self.inc_parser.get('one_atom')
+            # Calculations of relativistic quantum numbers are based
+            # on Table 6.6 in Wien2k userguide
+            # http://www.wien2k.at/reg_user/textbooks/usersguide.pdf
+            for atom_index, atom in enumerate(atoms):
+                kappas = atom.get('kappas')
+                n_quantum_numbers = atom.get('n_quantum_numbers')
+                occupancies = atom.get('occupancies')
+                number_of_orbitals = len(kappas)
+                for orbital in range(number_of_orbitals):
+                    max_occupancy = abs(kappas[orbital] * 2)
+                    occupancy = int(occupancies[orbital])
+                    if occupancy < max_occupancy:
+                        if kappas[orbital] >= 0:
+                            spin_quantum_number = -1
+                        else:
+                            spin_quantum_number = 1
+                        j_quantum_number = (
+                            -kappas[orbital] / spin_quantum_number - 1 / 2
+                        )
+                        l_quantum_number = j_quantum_number - (spin_quantum_number / 2)
+                        electrons_excited = max_occupancy - occupancy
+                        n_quantum_number = n_quantum_numbers[orbital]
+                        atom_par = sec_method.atom_parameters
+                        atom_obj = AtomParameters()
+                        atom_obj.atom_index = atom_index
+                        core_hole = CoreHole()
+                        atom_obj.core_hole = core_hole
+                        j_quantum_number = np.array([j_quantum_number])
```

Comment:
In the definition of j_quantum_number (see the link below), I found that shape should be [1..2], so I created an array. (Otherwise, tests failed on that.)  

https://nomad-lab.eu/prod/v1/gui/analyze/metainfo/nomad/section_definitions@nomad.datamodel.metainfo.simulation.method.AtomParameters/core_hole/j_quantum_number

Reply:
OK, fair enough